sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -202,6 +203,16 @@ public class Configs {
     //                + "\"applyDiagnosticThresholdsForTransportLevelMeters\":true}");
     public static final String METRICS_CONFIG = "COSMOS.METRICS_CONFIG";
     public static final String DEFAULT_METRICS_CONFIG = CosmosMicrometerMetricsConfig.DEFAULT.toJson();
+    private static final String DEFAULT_PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG = PartitionLevelCircuitBreakerConfig.DEFAULT.toJson();
+    private static final String PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG = "COSMOS.PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG";
+    private static final String STALE_COLLECTION_CACHE_REFRESH_RETRY_COUNT = "COSMOS.STALE_COLLECTION_CACHE_REFRESH_RETRY_COUNT";
+    private static final int DEFAULT_STALE_COLLECTION_CACHE_REFRESH_RETRY_COUNT = 2;
```

Comment:
I know we haven't done this consistently - but in Spark we have a description of configs added and that is super helpful. Maybe add a brief comment for each of the env variable settings here explaining what the setting is used for? Also do we need to have hard boundaries (lower or upper) to avoid that customers can completely shoot themselves in the foot?

Reply:
Fixed in next iteration. I am starting with lower bounds, setting extremely high values renders this feature as a no-op.